src/kernel/mconfig.ml

```diff
@@ -695,11 +618,6 @@ let ocaml_ignored_parametrized_flags = [
   "-zero-alloc-checker-details-cutoff";
   "-zero-alloc-checker-join";
   "-dgranularity";
-||||||| 7b73c6aa3f
-  "-error-style"; "-dump-dir";
-=======
-  "-error-style"; "-dump-dir"; "-cmi-file";
```

Comment:
I think we're missing `-cmi-file`?

Reply:
I don't think so. Merlin handles that flag now:

```
  ( "-cmi-file",
    Marg.param "file" (fun cmi_file ocaml ->
        {ocaml with cmi_file = Some cmi_file}),
    "<file>  Use the <file> interface to type-check"
  );

```

Reply:
Ah ok